buildSrc/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ plugins {
 
 dependencies {
     implementation(gradleApi())
-    implementation("com.android.tools.build:gradle:7.0.0-alpha08")
+    implementation("com.android.tools.build:gradle:7.0.0-beta01")
     implementation("org.jacoco:org.jacoco.core:0.8.5")
-    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.30")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.32")
```

Comment:
One of the Compose drawbacks is the compiler plugin. Really, more of a Kotlin Compiler Plugin drawback.. unstable apis, and Compose projects will be a little behind on Kotlin versions for the foreseeable future..